packages/terra-form-field/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "terra-form-field",
+  "main": "lib/Field.js",
+  "version": "0.0.0",
+  "description": "The Form Field component handles the layout of a standard form field.",
```

Comment:
Nit: This seems like it's saying "a form field is a form field". Can we use a variation the second part of the description above? Something that indicates the form field handles the text / node layout of a label, helper, and error?